Index: jni/Android.mk
===================================================================
--- jni/Android.mk	(revision 602)
+++ jni/Android.mk	(working copy)
@@ -88,6 +88,7 @@
 
 SILK     := silk
 LOCAL_MODULE    := silkcommon
+LOCAL_MODULE_FILENAME	:= silkcommon
 LOCAL_SRC_FILES :=  $(SILK)/src/SKP_Silk_A2NLSF.c \
 	$(SILK)/src/SKP_Silk_CNG.c \
 	$(SILK)/src/SKP_Silk_HP_variable_cutoff_FIX.c \
Index: jni/Application.mk
===================================================================
--- jni/Application.mk	(revision 602)
+++ jni/Application.mk	(working copy)
@@ -1,4 +1,4 @@
-APP_PROJECT_PATH := $(call my-dir)
+APP_PROJECT_PATH := $(call my-dir)/..
 APP_MODULES  := OSNetworkSystem
 APP_MODULES	 += speex_jni
 APP_MODULES	 += bv16_jni
